text/0000-new-buildpack-toml-keys.md

```diff
@@ -0,0 +1,88 @@
+# Meta
+[meta]: #meta
+- Name: New Buildpack Descriptor Keys
+- Start Date: 2020-12-12
+- Author(s): [@jkutner](@jkutner)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal for new keys in the `buildpack.toml` file.
+
+# Definitions
+[definitions]: #definitions
+
+* _buildpack descriptor_ - The `buildpack.toml` file that describes a buildpack.
+
+# Motivation
+[motivation]: #motivation
+
+New optional fields in the `buildpack.toml` would allow us to introduce more information into the Buildpack Registry search API and the web interface.
+
+# What it is
+[what-it-is]: #what-it-is
+
+* Add `description` as a free text field
+* Add `licenses` table
+
+# How it Works
+[how-it-works]: #how-it-works
+
+See the spec changes below for full details.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+- It introduces some none machine readable fields to an otherwise machine readable descriptor.
```

Comment:
```suggestion
- It introduces some non-machine-readable fields to an otherwise machine readable descriptor.
```
typo, but also, is this true? I find it confusing.

The fields can be parsed and printed automatically? It's no less machine-readable than the existing `metadata` table.

Reply:
yea, i couldn't think of the right way to describe these fields. What I mean is that they are structured in a way that a machine could do something with the value other than store it. I think every other value (except `name`) can be parsed/processed in a way that a machine can change it's behavior based on the values

Reply:
not `metadata` necessarily?

Reply:
addressed in 3ffe0a7